WordPress/Classes/Utility/CoreDataStack.h

```diff
@@ -9,8 +9,26 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)saveContextAndWait:(NSManagedObjectContext *)context;
 - (void)saveContext:(NSManagedObjectContext *)context;
 - (void)saveContext:(NSManagedObjectContext *)context withCompletionBlock:(void (^ _Nullable)(void))completionBlock onQueue:(dispatch_queue_t)queue NS_SWIFT_NAME(save(_:completion:on:));
+
+/// Execute the given block with a background context and save the changes.
+///
+/// This function _blocks_ its running thread. The changed made by the `aBlock` argument are saved before this
+/// function returns.
+///
+/// - Parameter aBlock: A closure which uses the given `NSManagedObjectContext` to make Core Data model changes.
```

Comment:
Nitpick for naming consistency.

```suggestion
/// - Parameter aBlock: A block which uses the given `NSManagedObjectContext` to make Core Data model changes.
```